src/rgw/rgw_sal_store.h

```diff
@@ -439,7 +439,18 @@ class StoreZone : public Zone {
 };
 
 class StoreLuaManager : public LuaManager {
+protected:
+  std::string _luarocks_path;
 public:
+  const std::string& luarocks_path() const override {
+    return _luarocks_path;
+  }
+  void set_luarocks_path(const std::string& path) override {
+    _luarocks_path = path;
+  }
```

Comment:
this needs to be thread-safe if you're writing it from one thread (the librados watcher callback) and reading it from others

Reply:
done in: 0fb13a74de751590c960078c8bcdaa82297910db